Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0-apache
+
+RUN apt-get update \
+ && apt-get install -y git zlib1g-dev \
+ && docker-php-ext-install zip \
+ && a2enmod rewrite \
+ && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/apache2.conf \
+ && mv /var/www/html /var/www/public \
+ && curl -sS https://getcomposer.org/installer \
```

Comment:
Do you actually need composer in the container? From what it looks like here, installation will be run outside the container anyway, no?